### DISCUSSION TITLE
Login Issues Discussion

### DISCUSSION THREAD

**Discussion by hjiangsu** [thunder-app/thunder#41] [General]
I have heard that a lot of people are having issues logging into particular instances. I'm creating this discussion as a place to talk about those issues to see if I can find out what the underlying cause of the issue is.

If you are having issues with login, please comment down below with the following information:
- Instance name and URL that you're using to login
- What error are you getting? Is there a specific error that pops up, or is it infinitely loading, etc.
- What version of the app are you currently using?
- Have you tried to login to a different instance successfully?

And any other information you think might be relevant!

**Comment by darukutsu** [comment]
**Instance name and URL that you're using to login**
https://infosec.exchange with disabled 2fa
I tried both url and simplified. Tried email and nickname login(without @)


**What error are you getting? Is there a specific error that pops up, or is it infinitely loading, etc.**
Whole app freezes, can't interact with anything. I need to restart it.
![Screenshot_20230630-172727_Thunder.png](https://github.com/hjiangsu/thunder/assets/54768193/459b7536-e9b0-417c-849f-dee2a2b55242)

**What version of the app are you currently using?**
0.2.1(8)

**Reply by hjiangsu** [reply]
Is this a lemmy instance? As of right now, only lemmy instances are supported!

**Reply by darukutsu** [reply]
I'm new to whole fediverse concept. It was in linked instances https://lemmy.ml/instances.

**Reply by Tealk** [reply]
The linked one is a Mastodon instance, this is currently not supported.
